searcher.proto

```diff
@@ -21,18 +21,34 @@ message SendBundleResponse {
 }
 
 message ProgramSubscriptionV0 {
+  // Base58 encoded program id that transactions mention
   repeated string programs = 1;
+
+  // Regions to subscribe to
+  // list of valid regions: https://jito-labs.gitbook.io/mev/systems/connecting/mainnet
+  repeated string regions = 2;
 }
 
 message WriteLockedAccountSubscriptionV0 {
+  // Base58 encoded account pubkey that transactions mention
   repeated string accounts = 1;
+
+  // Regions to subscribe to
+  // list of valid regions: https://jito-labs.gitbook.io/mev/systems/connecting/mainnet
+  repeated string regions = 2;
 }
 
 message MempoolSubscription {
+  // Filter by program id or account pubkey
   oneof msg {
     ProgramSubscriptionV0 program_v0_sub = 1;
     WriteLockedAccountSubscriptionV0 wla_v0_sub = 2;
   }
+
+  // Filters transactions to originate from specified regions.
+  // Defaults to the currently connected region.
+  repeated string subscribed_regions = 3;
```

Comment:
what's diff between this + regions in the ProgramSubscriptionV0 + WriteLockedAccountSubscriptionV0? imo just do one or the other unless something not covered there?

Reply:
fixed ty